images/router/haproxy/conf/haproxy-config.template

```diff
@@ -204,7 +204,7 @@ backend be_sni
 
 frontend fe_sni
   # terminate ssl on edge
-  bind 127.0.0.1:{{env "ROUTER_SERVICE_SNI_PORT" "10444"}} ssl no-sslv3 {{ if gt (len .DefaultCertificate) 0 }}crt {{.DefaultCertificate}}{{ else }}crt /var/lib/haproxy/conf/default_pub_keys.pem{{ end }} crt-list /var/lib/haproxy/conf/cert_config.map accept-proxy
+  bind 127.0.0.1:{{env "ROUTER_SERVICE_SNI_PORT" "10444"}} ssl no-sslv3 {{ if matchPattern "true|TRUE" (env "ROUTER_STRICT_SNI" "") -}} strict-sni {{ end }}{{ if gt (len .DefaultCertificate) 0 }}crt {{.DefaultCertificate}}{{ else }}crt /var/lib/haproxy/conf/default_pub_keys.pem{{ end }} crt-list /var/lib/haproxy/conf/cert_config.map accept-proxy
```

Comment:
It looks right to me, but I'd like to see some whitespace so I can follow what we are doing a little more clearly:

bind 127.0.0.1:{{env "ROUTER_SERVICE_SNI_PORT" "10444"}} ssl no-sslv3 
  {{- if matchPattern "true|TRUE" (env "ROUTER_STRICT_SNI" "") }} strict-sni {{ end }}
  {{- if gt (len .DefaultCertificate) 0 }} crt {{.DefaultCertificate}} {{ else }} crt /var/lib/haproxy/conf/default_pub_keys.pem {{ end }}
  {{- ""}} crt-list /var/lib/haproxy/conf/cert_config.map accept-proxy